src/features/unified-share-modal/__tests__/ContactRestrictionNotice.test.js

```diff
@@ -60,9 +60,9 @@ describe('features/unified-share-modal/ContactRestrictionNotice', () => {
 
     describe('render', () => {
         test('should render default ContactRestrictionNotice component', () => {
-            expect(wrapper.is('InlineNotice')).toBe(true);
-            expect(wrapper.find(`FormattedCompMessage[id="${pluralMessageId}"]`)).toHaveLength(1);
-            expect(wrapper.find('[data-resin-target="justificationReasonsSelect"]')).toHaveLength(1);
+            expect(wrapper.is('Tooltip')).toBe(true);
+            expect(wrapper.find('InlineNotice')).toHaveLength(1);
+            expect(wrapper.find('[data-resin-target="removeBtn"]')).toHaveLength(1);
```

Comment:
Minor: you could also use `wrapper.exists()` here

Reply:
Probably not an issue for this particular component, but I usually prefer `toHaveLength` in order to cover cases where there could be more than one component of the same kind and the wrong one could be matched by accident.